web/src/html.js

```diff
@@ -405,7 +421,14 @@ async function getHtml(ctx) {
     const referrerQuery = escapeHtmlProperty(getParameterByName('r', ctx.request.url));
 
     if (claim) {
-      const ogMetadata = await buildClaimOgMetadata(claimUri, claim, {}, referrerQuery);
+      const ogMetadata = await buildClaimOgMetadata(
+        claimUri,
+        claim,
+        {
+          userAgent: ctx.request.header['user-agent'],
```

Comment:
Will try to test it, but I assume `request` and `header` will always be present?
I recall some properties don't appear at times.

Reply:
I think only the first one is in a `try`, the rest aren't.  
Maybe just `try` and stash at the top, and re-use?

The reason I ask all this is just because of seeing `  if (ctx?.request?.url) {` ... it's either over-cautious, or null happened before.

Reply:
Good point. I didn't notice that line.